BrainPortal/app/views/resource_usage/_resource_usage_table.html.erb

```diff
@@ -98,8 +99,54 @@
                      :value => (@total.to_f / @resource_usages.total_entries.to_f)
                     )
                   ) %>)
+  </p>
   <% end %>
 </fieldset>
+<fieldset>
+  <legend>
+    Additional filtering
+    <%= show_hide_toggle "(show/hide)", ".filtration", :class => 'task_help_link', :slide_effect => true, :slide_duration => 'fast'  %>
+  </legend>
+
+
+  <%
+    is_hidden = (params[:data] && params[:data][:created_at].present?) ||
```

Comment:
This becomes too complex, maybe rewrite this with something like

```ruby
hidden=something here
if condition
  hidden = opposite of something
end
```

or even better, several

```ruby
hidden = initial
hidden = opposite if cond1
hidden = opposite if cond2
```